integration_testing/features/delete-multiple-items-permanently.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Delete multiple items permanently

```

Comment:
Unify this scenario with the one before previous into a single `delete-resources` .feature file with 2 scenarios. 

We are trying to keep the filenames simple, but do use the phrasing _single/multiple topics and/or resources_ in the scenario steps.